concordium-consensus/tests/e2e/Spec.hs

```diff
@@ -0,0 +1,21 @@
+module Main where
```

Comment:
I am not sure why this should be a separate test suite - doesn't it make sense for it to just combine with the consensus tests?

Reply:
I would also prefer that. It is in the style of other consensus tests.

Reply:
It's moved now.